### PR TITLE
Add crypto.py

### PR DESCRIPTION
**Pull request by AndroidKitKat**
Needs an API key from coinmarketcap.com

### CODE REVIEW

File: src/bobbit/modules/crypto.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        except KeyError:
+            response = 'No results'
+
+    return message.with_body(response)
+
+# Register
+
+def register(bot):
+    global API_KEY
+    config = bot.config.load_module_config('crypto')
```
Comment (AndroidKitKat) [inline src/bobbit/modules/crypto.py:61]: Is this format for getting the API Key right?

Comment (pbui) [reply src/bobbit/modules/crypto.py:61]: Yes

File: src/bobbit/modules/crypto.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# crypto.py
+
+# Metadata
+
+NAME = 'crypto'
+ENABLE = True
+PATTERN = '^!crypto (?P<symbol>.*$'
```
Comment (pbui) [inline src/bobbit/modules/crypto.py]: Need missing ): `'^!crypto (?P<symbol>.*)$'`

Comment (AndroidKitKat) [reply src/bobbit/modules/crypto.py]: oic

**Pull request merged**